API/controllers/channelController.js

```diff
@@ -0,0 +1,108 @@
+const express = require('express');
+const router = express.Router();
+const mongoose = require('mongoose');
+const _ = require('lodash');
+
+require('../models/Channels');
+const Channel = mongoose.model('Channel');
+
+/**
+ * Methods:
+ *  post(/) => Creates new channel
+ *  get(/:id) => Gets channel by id
+ *  get(/:id/posts) => Gets all posts associated with channel's tags
+ *  put(/:id) => Updates channel with given id
+ *  delete(/:id) => Deletes channel with given id
+ */
+
+
+/**
+ * Create a new channel
+ */
+router.post('/', (req, res) => {
+  Channel.create(req.body).then(channel => {
+    res.json(200, channel);
+  })
+  .catch(err => {
+    if (typeof err === 'object') {
+      // Actual runtime error
+      res.json(500, err.message);
+    }
+    else {
+      // Purposefully did not complete action, returned message
+      res.json(400, err);
```

Comment:
Should change status 400 to 403 (400 -> invalid, 403 -> understood but forbidden)